src/app/(proper_react)/layout.tsx

```diff
@@ -3,15 +3,22 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { ReactNode } from "react";
+import Script from "next/script";
 import { getL10nBundles, getLocale } from "../functions/server/l10n";
 import { L10nProvider } from "../../contextProviders/localization";
 import { ReactAriaI18nProvider } from "../../contextProviders/react-aria";
+import { getNonce } from "../(nextjs_migration)/functions/server/getNonce";
 
 export default function Layout({ children }: { children: ReactNode }) {
   const l10nBundles = getL10nBundles();
 
   return (
     <L10nProvider bundleSources={l10nBundles}>
+      <Script
+        type="module"
+        src="/nextjs_migration/client/js/analytics.js"
```

Comment:
Hey @flozia, I haven't looked at this PR in detail yet, but I'm a bit worried about this line. In any case I'd want to avoid references to `nextjs_migration` (I'd like to delete that dir after we've fully transitioned), but also I'd like to avoid vendoring our dependencies, and having implicit global variables.

Then again, this might only be temporary, what with Glean and all, @rhelmer?

But otherwise, maybe we could look into using something like [react-ga4](https://www.npmjs.com/package/react-ga4), or update [react-ga](https://www.npmjs.com/package/react-ga) ourselves (since that's a Mozilla package) to work with GA4?

Reply:
We can use one of the proposed packages. I’m not sure about what events we’ll be using Glean and GA for. Before pushing this PR up [I asked about this](https://mozilla.slack.com/archives/C04BLQVN151/p1694452369102829)and it sounded like we are going to utilize both @rhelmer?

Reply:
Yeah, I believe both at first, and then we can remove GA later after we've verified that they report similar data? Probably worth it to create a wrapper that reports to both, so we only need to update it there down the road.

Reply:
Yes I think we'll use both at least for the foreseeable future - having one tool that did event collection both on the back-end and front-end and let us look at it as a single event stream during analysis (and even better had easy-to-use analysis tools) would be the ideal, but we don't have anything like that available to us currently. Glean is a requirement for Nimbus so we also need it for that reason, but I think the analysis tools GA (even 4) gives us is pretty hard to do without.

I like the idea of having a wrapper that reports to both - there's an extra step with Glean where we need to record and have approved the metrics we're collecting, but I don't think that's necessarily a bad thing.

Reply:
Yeah if the events here are only going to be sent to one of the two, creating an abstraction over them seems out of scope.

Reply:
Replaced with the `gaHook` that has been introduced by PR #3440 in https://github.com/mozilla/blurts-server/pull/3402/commits/4673a21723c9c723585b6f9b08628b7b7126d65d.